src/utils/i18n.ts

```diff
@@ -24,6 +24,22 @@ i18next.use(I18nextBrowserLanguageDetector).init({
         'Postal Code': '郵便番号',
       },
     },
+    pt-BR: {
```

Comment:
I think we need to replace this code from `pt-BR` to `'pt-BR'` to resolve this error:
https://github.com/wpkyoto/stripe-pwa-elements/actions/runs/5132459150/jobs/9255449130#step:5:35

Reply:
Hi Kamoto,

Pull request made according to your guidelines.

Thanks